### ISSUE TITLE
Tundra parts are shifting off their attachment point

### ISSUE THREAD

**Issue by Alshain01** [UmbraSpaceIndustries/MKS#1313]
I've had this happen many times now.  The base was assembled with KIS and at the time of save, everything was fine.  However when the game loads, they seem to be attached where they shouldn't be.  

![screenshot104](https://user-images.githubusercontent.com/4256578/28803422-09383d0a-7622-11e7-93c4-39e22ddf8e94.png)

![screenshot103](https://user-images.githubusercontent.com/4256578/28803423-09559e04-7622-11e7-8529-c03928bf0b0f.png)

![screenshot105](https://user-images.githubusercontent.com/4256578/28803452-42f8b3f8-7622-11e7-8ea2-0d9229922292.png)

Ground tether is attached, however despite that my base is slowly sliding down hill in the direction of these modules.  That may (or may not) have something to do with it.  When I built my base it was right next to those two ships back  in the back.

**Comment by Prototype2001** [comment]
I can confirm this, at first i thought it was just the weight of one craft thats causing the shift but then noticed weight had no factor. http://imgur.com/a/gShc5#XSqsRjf second picture is my experience with this.

**Comment by BobPalmer** [comment]
Is this happening with parts not attached via KIS?

**Comment by sh1pman** [comment]
This happens to me all the time. In fact, this is my biggest issue with base building at the moment. And I'm 99% sure it happens with parts not attached via KIS.

**Comment by Prototype2001** [comment]
For me it seems to be occurring via KIS attachments, the attachments don't need to be a docking of two vessels. Just a simple flexo-tube or karbonite converter or docking port sr, all become severely offset. I believe this is tied to all or some tundra modules, I've had normal docking of two crafts via KIS docking port sr when the port was KIS attached to a non tundra item.

**Comment by sh1pman** [comment]
It's not limited to Tundra parts. I've seen this happening to 3.75m reactors and round kontainers. Honestly, I think it's a stock issue, not an MKS one.

**Comment by BobPalmer** [comment]
Does it only occur when ground tether is turned on?

**Comment by sh1pman** [comment]
No, turning ground Tether on or off has no effect on it. The only thing that seems to matter is the mass: the heavier the part, the stronger the distortion. Autostrut doesn't help either: the shift only happens when the vessel (or the scene) is reloaded. You can usually tell if a newly-attached part is going to be shifted by timewarping - during the timewarp its position will temporarily shift, but reset to normal when you go back to 1x timewarp. Upon scene reloading the position of said part will be permanently shifted. 

Complex bases with long and branched structure seem to be more affected by this glitch. It also seems to be more pronounced on high-gravity planets like Eve.

I hope these observations will help.

**Comment by ravensierra** [comment]
Has this issue been resolved in 1.1? Any assistance I can provide with logs etc otherwise?
